Content.Server/PAI/PAISystem.cs

```diff
@@ -117,5 +119,30 @@ public void PAITurningOff(EntityUid uid)
             if (proto != null)
                 _metaData.SetEntityName(uid, proto.Name);
         }
+    // SS220-Add-Languages begin
+        if (TryComp<LanguageComponent>(uid, out var languages))
+        {
+            languages.LearnedLanguages.Clear();
+            languages.LearnedLanguages.Add(LanguagesPrototype.Galactic);
+            languages.LearnedLanguages.Add(LanguagesPrototype.Universal);
+            languages.CurrentLanguage = languages.LearnedLanguages[0];
+        }
+    }
+
+    public void GetUserLanguages(EntityUid? ent, EntityUid pai)
```

Comment:
```suggestion
    public void CopyUserLanguages(EntityUid? ent, EntityUid pai)
```

Имхо: мб этот метод стоит закинуть в систему языков